<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyListView</class>
 <widget class="QWidget" name="MyListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="pic_list_view">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>871</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_pic_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_pic_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_all_pic_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除全部</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
